### PR TITLE
Bug 1852301: Add integration test for Utilization and Events

### PR DESCRIPTION
**Pull request by gouyang**
Implementation for https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=CNV-3331 and https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=CNV-3329.

### CODE REVIEW

**Comment by gouyang** [comment]
@suomiy Could you please take a look?

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.tab.dashboard.scenario.ts [hunk 1]
```diff
@@ -54,46 +53,101 @@ describe('Test VM dashboard', () => {
...
 
   it('ID(CNV-3332) Details card', async () => {
-    expect(vmDetailsName.getText()).toEqual(vm.name);
-    expect(vmDetailsNamespace.getText()).toEqual(vm.namespace);
-    expect(vmDetailsNode.getText()).not.toEqual(NOT_AVAILABLE);
-    expect(vmDetailsIPAddress.getText()).not.toEqual(NOT_AVAILABLE);
+    await browser.wait(waitForStringInElement(dashboardView.vmDetailsHostname, vm.name));
+    expect(dashboardView.vmDetailsName.getText()).toEqual(vm.name);
+    expect(dashboardView.vmDetailsNamespace.getText()).toEqual(vm.namespace);
+    expect(dashboardView.vmDetailsNode.getText()).not.toEqual(NOT_AVAILABLE);
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.tab.dashboard.scenario.ts:95]: can we test the hostname as well?

Comment (gouyang) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/vm.tab.dashboard.scenario.ts:95]: the hostname is tested above within `await`, the installation of guest agent takes very long time, the monitor of guest agent is in `Status card`, even the alert message disappeared, the hostname may not shows immediately, hence use `await` for it.

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/vm.tab.dashboard.scenario.ts:95]: ah okay, I am sorry I missed that

**Pull request merged**